Updated getting started with step by step instructions (#922)
conan-io/docs: getting_started.rst

   :width: 500 px
   :align: center

For example, the command :command:`conan install . --settings os="Linux" --settings compiler="gcc"`, performs these steps:

- Checks if the package recipe (for ``Poco/1.9.0@pocoproject/stable`` package) exists in the local cache. If we are just starting, the
  cache is empty.
- Looks for the package recipe in the defined remotes. Conan comes with `conan-center`_ Bintray remote as the default, but can be changed.
- If the recipe exists, the Conan client fetches and stores it in your local cache.

> - With the package recipe and the input settings (Linux, GCC), Conan looks for the corresponding binary is in the local cache.

Reviewer: ```suggestion
- With the package recipe and the input settings (Linux, GCC), Conan looks for the corresponding binary in the local cache.
```

- Then Conan searches the corresponding binary package in the remote and fetches it.

> - Finally, it generates an appropriate file for the build system specified the ``[generators]`` section.

Reviewer: ```suggestion
- Finally, it generates an appropriate file for the build system specified in the ``[generators]`` section.
```

There are binaries for several mainstream compilers and versions available in Conan Center repository in Bintray, such as Visual Studio 14,
15, Linux GCC 4.9 and Apple Clang 3.5... Conan will throw an error if the binary package required for specific settings doesn't exist. You

> can try to build the binary package from sources using :command:` conan install --build=missing`. You will find more info

Reviewer: ```suggestion
can build the binary package from sources using :command:` conan install --build=missing`, it will succeed if your configuration is supported by the recipe. You will find more info
```

the :ref:`getting_started_other_configurations` section.

.. attention::

    When a GCC **compiler >= 5.1** is detected, the setting modeling for the c++ standard library is set as follows: The ``compiler.libcxx``
    is set to ``libstdc++`` that represents the old ABI compatibility for better compatibility. Your compiler default is most likely to be

>     set to the new ABI, so you might want to change it to ``libstdc++11`` to use the new ABI compliant with CXX11 directives and run

Reviewer: ```suggestion
    set to the new ABI, so you might want to change it to ``libstdc++11`` in the definition of the default profile to use the new ABI compliant with CXX11 directives and run
```

.. code-block:: bash

>     $ conan install . --settings arch=x86

Reviewer: (Look at the text line explaining this command) What if the default was a x86 architecture? I would use something related to the *gcc_x86* profile already mentioned:

```
conan install . --profile gcc_x86 --settings arch=x86_64
```

Reviewer: Also, in the line below, use the same param, not the short one: ``--settings arch=****``
